go-controller/pkg/ovn/master.go

```diff
@@ -687,7 +687,16 @@ func (oc *Controller) syncGatewayLogicalNetwork(node *kapi.Node, l3GatewayConfig
 	}
 
 	if l3GatewayConfig.NodePortEnable {
-		err = oc.handleNodePortLB(node)
+		gatewayRouter := types.GWRouterPrefix + node.Name
+		var physicalIPs []string
+		if physicalIPs, _ = oc.getGatewayPhysicalIPs(gatewayRouter); physicalIPs == nil {
```

Comment:
So... we read the gateway physical IPs but then we never use them for anything? Do we need this code?